sympy/polys/densebasic.py

```diff
@@ -1841,6 +1841,23 @@ def dmp_slice(f, m, n, u, K):
     return dmp_slice_in(f, m, n, 0, u, K)
 
 
+def dup_truncate(f, n, K):
+    """
+    Truncate ``f`` to the first ``n`` terms in ``K[x]``.
+
+    Examples
+    ========
+
+    >>> from sympy.polys.domains import ZZ
+    >>> from sympy.polys.densebasic import dup_truncate
+
+    >>> dup_truncate([1, 2, 3, 4, 5], 3, ZZ)
+    [3, 4, 5]
```

Comment:
The inputs should be ZZ(1), ZZ(2) etc. You can use `ZZ.map` though.

These doctests would be more understandable if they displayed outputs that look like polynomials. There is no `dup_print` function but if there was it could be:
```python
>>> p = ZZ.map([1, 2, 3, 4, 5])
>>> p3 = dup_truncate(p, 3, ZZ)
>>> dup_print(p, ZZ)
x**4 + 2*x**3 + 3*x**2 + 4*x + 5
>>> dup_print(p3, ZZ)
3*x**2 + 4*x + 5
```
Perhaps even `ZZ.map` could be replaced with `p = dup_from_list([1, 2, 3, 4, 5], ZZ)`